ols/src/llms/llm_loader.py

```diff
@@ -48,11 +56,15 @@ def __init__(
         """Initialize loader using provided provider, model, and other parameters."""
         self.logger = logger if logger is not None else Logger("llm_loader").logger
         if provider is None:
-            raise Exception("ERROR: Missing provider")
+            msg = "ERROR: Missing provider"
```

Comment:
I know it's out of the scope of this PR, but having "ERROR" there is redundant, right?

Reply:
Updated too